examples/rsp-cra-18/src/sections/ContentExamples.tsx

```diff
@@ -0,0 +1,40 @@
+import {Flex, Divider, Avatar, Content, Footer, Header, Heading, IllustratedMessage, Image, Keyboard, Text, View, TextField, Well} from '@adobe/react-spectrum';
+import NotFound from '@spectrum-icons/illustrations/NotFound';
+
+export default function ContentExamples() {
```

Comment:
This one is a little weird or too comprehensive for me because components like Content, Footer, Header, Heading, and Divider sort of work on their own, but we care more about them in context, like within a Dialog. And testing View feels sort of like testing Flex or Grid.

Reply:
yeah i see what you mean. for context, i basically just took what we had in the next.js example app and threw it into here but i think if i have the time to work on this again then making them reflect more "real-world" examples would be better